server/connection/socket.js

```diff
@@ -94,6 +94,14 @@ function msgSender(identify, isBuyer, message) {
     resolve('succ');
   });
 }
+
+async function plusChatRoomCountWhenFirstChat(room, productId) {
+  const roomInfo = await getReciveChatRoomInfo(room);
+  if (roomInfo.sellerin === 0 && roomInfo.buyerin === 0) {
+    await plustProductChatRoomCount(productId);
+  }
+}
```

Comment:
roomInfo를 사용하는 코드가 없어 객체분할 할당을 쓰면 좀 더 나아보입니다 :)

```suggestion
async function plusChatRoomCountWhenFirstChat(room, productId) {
  const {sellerin, buyerin} = await getReciveChatRoomInfo(room);
  if (sellerin === 0 && buyerin === 0) {
    await plustProductChatRoomCount(productId);
  }
}
```